src/fights.ts

```diff
@@ -1762,6 +1837,15 @@ const freeRunFightSources = [
       ],
     }
   ),
+  // Try to accelerate the shadow nc, if you're able to do a quest
+  new FreeRunFight(
+    () =>
+      have($item`closed-circuit pay phone`) &&
+      get("rufusQuestType") !== "items" &&
+      !have($effect`Shadow Affinity`) &&
+      get("encountersUntilSRChoice") > 0,
+    (runSource: ActionSource) => garboAdventure(bestShadowRift(), runSource.macro)
+  ),
```

Comment:
Everything works as intended, except this code rarely runs as we grab shadow forest items in dailyItems, usually leaving us with turns of Shadow Affinity.

Not sure what I should do with this, since if you run garbo again after shadow affinity's run out, then this will trigger.
```suggestion
```